src/renderer/marketplace/notarization/common/message-container.jsx

```diff
@@ -0,0 +1,22 @@
+import React, { PureComponent } from 'react';
+import { connect } from 'react-redux';
+import { withStyles } from '@material-ui/core';
+import NotarizationMessageWidget from './message-widget';
+
+const styles = theme => ({});
+
+class NotarizationMessageContainer extends PureComponent {
+	render() {
+		const { onBackClick, messages } = this.props;
+		return <NotarizationMessageWidget onBackClick={onBackClick} messages={messages} />;
+	}
+}
+
```

Comment:
Simple functional components should use functions instead of classes for simplicity, see https://reactjs.org/docs/components-and-props.html#function-and-class-components.

Unless you believe this component will in the future need complex lifecycle functionality and/or local state

Reply:
@andregoncalves @designhorf it's better to keep it as class because it's a container and our version of react does not support hooks

A better question would be "Should this be a container?"